### DISCUSSION TITLE
Dynamic Routes only load client-side on Zeit Now

### DISCUSSION THREAD

**Discussion by BjoernRave** [vercel/next.js#11002] [Help] [answered]
Hey, my dynamic routes work fine locally, but when I deploy my project to Zeit Now, server-side rendering of my dynamice routes doesn't work and I get the `404: NOT_FOUND` screen from Now. 

They work however when accessed through the client-side.

I will be happy to provide additional info.

**Comment by Timer** [comment]
Please provide a reproducible demo so we can take a look—a deployment URL would work, too.

**Reply by BjoernRave** [reply]
okay, I will write you some test credentials + url on twitter so you can login and see it.

**Comment by BjoernRave** [comment]
@Timer Hey, any news? I sent you a message on twitter, with a login for the deployment. I hope that's sufficient to investigate.

**Comment by BjoernRave** [comment, marked as answer]
I just checked again and it's fixed now, not sure why :D
